### PR TITLE
[CORE-1049] Contacts.list sort on date_updated

### PR DESCRIPTION
**Pull request by JordyDeGroeve**

### CODE REVIEW

**Review by WouterSioen — DISMISSED** [review]

File: apiary.apib [hunk 1]
```diff
@@ -608,9 +608,11 @@ Get a list of contacts.
                 + field (enum[string], required)
                     + Members
                         + name - Sorts alphabetically on last_name and first_name
+                        + date_updated - Sorts on date_update
```
Comment (WouterSioen) [inline apiary.apib]: Wait, shouldn’t this be “updated_at”?

**Review by WouterSioen — APPROVED** [review]

**Review by dietrichm — APPROVED** [review]

File: apiary.apib [hunk 2]
```diff
@@ -608,9 +608,11 @@ Get a list of contacts.
                 + field (enum[string], required)
                     + Members
                         + name - Sorts alphabetically on last_name and first_name
+                        + updated_at - Sorts on date updated
```
Comment (WouterSioen) [inline apiary.apib:611]: I would've updated the comment too

File: apiary.apib [hunk 3]
```diff
@@ -608,9 +608,11 @@ Get a list of contacts.
...
                 + order (enum[string], optional)
                     + Members
                         + asc
+                        + desc
```
Comment (jenssegers) [inline apiary.apib:615]: This means that you can now also sort `desc` on `name`?

Comment (JordyDeGroeve) [reply apiary.apib:615]: Correct. This is already handled in the search. Was a small addition.

**Review by mrtus — APPROVED** [review]

**Review by rathesDot — APPROVED** [review]

**Pull request merged**